.github/workflows/docs.yml

```diff
@@ -5,6 +5,13 @@ on:
   push:
     paths:
       - "docs/**"
+      - ".github/workflows/docs.yml"
+    branches:
+      - main  # Set a branch to deploy
+  pull_request:
+    paths:
```

Comment:
also lets use `dorny/paths-filter` and remove the conditionals here, then we can make the job a required one if case we change any docs

Reply:
I'm not sure I understand. Are you saying you would like the hugo build to always run on every change, but only publish if the docs or workflow file changes?